RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -450,11 +454,12 @@ SiPixelCluster PixelThresholdClusterizer::make_cluster(const SiPixelCluster::Pix
            ++r) {
         if (theBuffer(r, c) >= thePixelThreshold) {
           SiPixelCluster::PixelPos newpix(r, c);
-          if (!acluster.add(newpix, theBuffer(r, c)))
+          auto const newpix_adc = std::min(theBuffer(r, c), int(std::numeric_limits<uint16_t>::max()));
+          if (!acluster.add(newpix, newpix_adc))
             goto endClus;
```

Comment:
It was ages I didn't see a `goto` statement in a C++ code...

Reply:
There is an issue about it (for a while now) https://github.com/cms-sw/cmssw/issues/34705